tools/ci_build/github/linux/docker/build_scripts/README.md

```diff
@@ -7,13 +7,22 @@ diff -r /data/bt/os/manylinux/docker/build_scripts/install-entrypoint.sh ./insta
 > yum versionlock cuda* libcudnn*
 \ No newline at end of file
 diff -r /data/bt/os/manylinux/docker/build_scripts/install-runtime-packages.sh ./install-runtime-packages.sh
-87c87,92
-<       TOOLCHAIN_DEPS="devtoolset-9-binutils devtoolset-9-gcc devtoolset-9-gcc-c++ devtoolset-9-gcc-gfortran"
+87c87,95
```

Comment:
Need resolve merge here?

Reply:
It's a readme file. To note what I changed.